common/hdr-model.h

```diff
@@ -0,0 +1,80 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <rsutils/json.h>
+#include <librealsense2/rs.hpp>
+#include "ux-window.h"
+
+namespace rs2 {
+struct hdr_config
+{
+    struct control_item
+    {
+        int DepthGain;
+        int DepthManExp;
+
+        control_item();
+        control_item( int gain, int exp );
+    };
+
+    struct preset_item
+    {
+        int iterations;
+        std::vector< control_item > controls;
+
+        preset_item();
+    };
+
+    struct sub_preset
```

Comment:
Avoid using "sub_preset". It is an internal FW name that have no meaning to us or to users.
What is the use here - frame_config? hdr_configuration? Maybe we don't need it as an inner struct but can be the outer struct fields